src/resources/postcss/tickets-commerce/admin/orders/single.pcss

```diff
@@ -0,0 +1,286 @@
+/**
+ * Event Tickets - Single Admin Order Page
+ *
+ * @since TBD
+ */
+
+body.wp-admin.post-type-tec_tc_order {
+	@media screen and (max-width: 782px) {
+		.tribe-mobile-hidden {
+			display: none;
+		}
+	}
+
+	@media screen and (max-width: 1100px) {
+		.tribe-desktop-only {
+			display: none;
+		}
+	}
+
+	.tribe-mobile-only {
+		display: none;
+
+		@media screen and (max-width: 782px) {
+			display: inline-block;
+		}
+	}
```

Comment:
We have some viewport vars defined in `_media-queries-css` let's use them where applicable.